app/models/review.rb

```diff
@@ -0,0 +1,13 @@
+class Review < ApplicationRecord
+  belongs_to :art
+  belongs_to :user
+
+  validates :rating, presence: true, inclusion: 1..5, numericality: { only_integer: true }
+  validate :user_has_booked_before
+
+  private
+
+  def user_has_booked_before
+    return true if user.bookings.include?(art)
+  end
```

Comment:
user.bookings is an array of Booking instances so it will never include art which is a Art instance.
You need to do `user.bookings.map(&:art).include?(art)` instead.

There is no need to return `true` if ...
You can just return if ...

Finally you need to add an error to this validation in case the validation doesn't succeed.
Try something like this : `errors.add(:user, "cannot review a Art he has never booked")`